### ISSUE TITLE
Optimism Support

### ISSUE THREAD

**Issue by arielman87** [0xsequence/public-roadmap#6]
### TL;DR:
Sequence will soon deploy indexers and relayers for Ethereum-compatible networks, and thereby offer their support inside of the wallet as well.
![Optimism status](https://user-images.githubusercontent.com/97196974/179073040-d9c34580-a622-4bd6-9ea4-7169600bbaf1.jpeg)

#### Check the [production status](https://status.sequence.info/?env=production) 

You will be able to use a variety of methods to query **Optimism chain ID = 10**
```ts
const wallet = sequence.getWallet()

console.log('chainId:', await wallet.getChainId())

const provider = wallet.getProvider()
console.log('provider.getChainId()', await provider.getChainId())

const signer = wallet.getSigner()
console.log('signer.getChainId()', await signer.getChainId())
```

### Official Channels:
**Twitter**: https://twitter.com/0xsequence 
**Discord**: https://discord.gg/sequence 
**Reddit**: https://reddit.com/r/0xSequence/
